methods/espresso/5.4.0.yml

```diff
@@ -0,0 +1 @@
+!include "methods/espresso/5.2.1.yml"
```

Comment:
Could we consider a better approach, where we by default use the latest previous version that's present here without having to define the yml file for each version we support?? For example, the 6.0.0 will look for its version, if not found - look for the closest previous version and grab it's yml intead.

Reply:
In my opinion, the Yaml files should be as explicit as possible. So searching backward for other versions should be done in code, for instance at the point where we construct the list of filters (`getFilterObjects()`).
Is suggest the following:
- adjustment of `getFilterObjects()` to cycle backward through versions if a version is not found
- deletion of duplicating Yaml files

Reply:
Seems like a lot of duplication, though. If/when we add a new version, we have to add a yml file here when no changes are expected. Why add what we can avoid?

Reply:
Apologies, perhaps I misunderstood your first comment and I think we both mean the same thing. With my suggestion above, the Yaml files would be reduced to the smallest set. I.e. if `methods/espresso/6.0.0.yml` does not exist, the respective key `"6.0.0"` would be missing in the final filter tree. Then when the filter objects are created, we can add a check for the version key so that a previous version is used.

Reply:
Files with single `!include ...` have been removed.